setup.py

```diff
@@ -227,7 +227,7 @@ def write_version(filename: str = os.path.join(*[my_dir, "airflow", "git_version
     'cassandra-driver>=3.13.0,<4',
 ]
 celery = [
-    'celery~=5.1,>=5.1.2',
+    'celery~=5.2,>=5.2.3',
```

Comment:
Can this not continue to allow Celery 5.1.x by declaring `celery>=5.1` instead?

Reply:
we usually only support the latest stable version in celery & https://github.com/celery/celery/releases/tag/v5.2.2 is not back ported.

Reply:
IF this one contains some security issue, then I am ok with >=5.2.3. 

But I thought a bit and we should really remove the upper limit - this is our default behaviour and our constraint mechanism will keep it in check but we will avoid any need for future changes like that for 5.3 and above.